content/docs/index.md

```diff
@@ -12,3 +12,4 @@ page_template = "docs-page.html"
     * [ACL](/topics/acl/)
 * Valkey Manual
   * [Keyspace Notifications](/topics/keyspace/)
+* [Release Notes](topics/release-notes/)
```

Comment:
humm. I don't mind the idea of linking the release notes but `/topics/release-notes/` isn't a documentation topic in valkey-doc

Reply:
Hi @stockholmux ! Thanks for reviewing this!  When I created this PR it was in combination with https://github.com/valkey-io/valkey-doc/pull/171 . If https://github.com/valkey-io/valkey-doc/pull/171 is not going to be included, then we can close this one. Hope this helps